src/apdu_sign.c

```diff
@@ -124,6 +124,7 @@ size_t baking_sign_complete(bool const send_hash, volatile uint32_t *flags) {
         case MAGIC_BYTE_BLOCK:
         case MAGIC_BYTE_BAKING_OP:
             guard_baking_authorized(&G.parsed_baking_data, &global.path_with_curve);
+            ux_empty_screen();
```

Comment:
`perform_signature` could THROW an error. Do we want the screen to blank in that case?

Reply:
Changed the baking_sign_complete return flow, to make sure ux_empty_screen() is called after successful perform_signature.